src/main/resources/include/hail/NativeObj.h

```diff
@@ -0,0 +1,63 @@
+#ifndef HAIL_NATIVEOBJ_H
+#define HAIL_NATIVEOBJ_H 1
+//
+// Declarations needed for C++ code to be integrated with Hail 
+//
+// Richard Cownie, Hail Team, 2018-04-03
+//
```

Comment:
I think we should leave of the leading and trailing // lines to save vspace.  The style guide doesn't comment on this specifically but it is consistent with all the examples in the style guide.